### ISSUE TITLE
terraform > 0.12.x not supported.

### ISSUE THREAD

**Issue by varunchandak** [avantoss/vault-infra#16]
I am using `Terraform v0.12.12`. When I am trying to build using this, I get an error:

```
➜  main git:(master) ✗ terraform init
Initializing modules...
- vault in ../modules/vault

Error: Unsupported Terraform Core version

This configuration does not support Terraform version 0.12.12. To proceed,
either choose another supported Terraform version or update the root module's
version constraint. Version constraints are normally set for good reason, so
updating the constraint may lead to other errors or unexpected behavior.


Error: Unsupported Terraform Core version

Module vault (from "../modules/vault") does not support Terraform version
0.12.12. To proceed, either choose another supported Terraform version or
update the module's version constraint. Version constraints are normally set
for good reason, so updating the constraint may lead to other errors or
unexpected behavior.
```

**Comment by lingrino** [comment]
Hi @varunchandak terraform `0.12+` includes major updates that are not backwards compatible. I currently have a PR to merge support for `0.12` and then I will tag that release as `2.0.0`. I'm hoping that will be merged and ready by tomorrow.

Thanks!

**Comment by varunchandak** [comment]
awesome... also would like to mention few things in your readme that I couldn't understand. Specifically point no. 9 and 13.

Or would be grateful if you could create a video demo for us to understand ?

**Comment by lingrino** [comment]
@varunchandak The PR was merged and the `2.0.0` tag was just pushed. `master` should now support `0.12+`.

Step 9 is asking you to upload certs that you have already generated on your own the the resources bucket. Some more information is in the [Resources Bucket Section](https://github.com/avantoss/vault-infra#s3-resources-bucket).

Step 13 is a one-time command that you need to run on **one** of the vault nodes that you have started. The Vault docs for that are [here](https://www.vaultproject.io/docs/commands/operator/init.html).

We will not be creating a video demo.
